isis/src/control/objs/BundleUtilities/AbstractBundleObservation.cpp

```diff
@@ -317,6 +300,8 @@ namespace Isis {
  */
 QString AbstractBundleObservation::formatBundleOutputString(bool errorPropagation, bool imageCSV) {
   // different for both
+  // TODO: either remove or update. 
+  return "Test";
```

Comment:
Do you think it's at all realistic to have something working for this at the parent class level, or should I just pull these out of this class?

Reply:
Same as CSV output, we can just make these pure virtual